src/main/java/teammates/ui/controller/InstructorCourseEditPageAction.java

```diff
@@ -32,26 +32,26 @@ public ActionResult execute() throws EntityDoesNotExistException {
         
         /* Setup page data for 'Edit' page of a course for an instructor */
         InstructorCourseEditPageData data = new InstructorCourseEditPageData(account);
-        data.course = courseToEdit;
+        
+        List<InstructorAttributes> instructorList = new ArrayList<InstructorAttributes>();
+        int offset = -1;
```

Comment:
offset is not really an index. It's more like a starting point. If it is -1 then show all instructors, otherwise, show the instructor at the position __offset__.